onnx/defs/rnn/defs.cc

```diff
@@ -60,6 +68,15 @@ std::function<void(OpSchema&)> RNNDocGenerator(const char* /*name*/) {
         "Must be one of forward (default), reverse, or bidirectional.",
         AttributeProto::STRING,
         std::string("forward"));
+    schema.Attr(
+        "batch_major",
+        "The shape format of the input X and output Y. "
+        "If 0, the shapes are [seq_length, batch_size, input_size] and "
+        "[seq_length, num_directions, batch_size, hidden_size] respectively."
+        "If not 0, the shapes are [batch_size, seq_length, input_size] and "
+        "[batch_size, num_directions, seq_length, hidden_size] respectively.",
+        AttributeProto::INT,
+        static_cast<int64_t>(1));
```

Comment:
I think that the default-value should have same behavior as previous version (giving us better compatibility). Shouldn't that be the value 0?

Reply:
Right, this was a mistake. Initially the attribute was called `time_major`, so its default value did replicate the old behaviour. Then I've changed it to `batch_major` but didn't update the default here (but only in the shape inference). Fixed.